src/modules/Channel/components/Message/index.tsx

```diff
@@ -33,8 +33,7 @@ type MessageUIProps = {
   hasSeparator?: boolean;
   chainTop?: boolean;
   chainBottom?: boolean;
-  handleScroll?: () => void;
-  handleMessageListHeightChange?: () => void;
+  handleScroll?: (isAffectedOnlyGround?: boolean) => void;
```

Comment:
isAffectedOnlyGround is not really clear, lets improve the name?

Reply:
how about `isBottomMessageAffected`? is it more clear? or any suggestion?

Reply:
both sounds okay